### PR TITLE
improve `rw upgrade` command and remove "publish PR packages" option

### PR DESCRIPTION
**Pull request by dac09**
## What does this do?
As per @peterp's suggestion, this implements the upgrade command to:
1. query npm to find the lastest/ canary package versions
2. iterate over a user's package.json (root, web, api) files, replace the versions of packages that start with @redwoodjs/*
3. run yarn install

What it looks like: https://s.tape.sh/UFLppMbI

### Note:
- I've removed the upgrade pr command/flag as its unstable

### CODE REVIEW

File: packages/cli/src/commands/upgrade.js [hunk 1]
```diff
@@ -28,12 +33,12 @@ export const builder = (yargs) => {
       type: 'string',
       coerce: validateTag,
     })
-    .option('pr', {
-      description: 'Installs packages for the given PR',
-      requiresArg: true,
-      type: 'string',
+    .option('verbose', {
+      alias: 'vvv',
```
Comment (peterp) [inline packages/cli/src/commands/upgrade.js:37]:
```suggestion
      alias: 'v',
```

File: packages/cli/src/commands/upgrade.js [hunk 2]
```diff
@@ -260,3 +128,124 @@ export const handler = async ({ d, tag, pr }) => {
...
+          updatePackageJsonVersion(basePath, ctx.versionToUpgradeTo, options),
+        skip: () => !fs.existsSync(pkgJsonPath),
+      }
+    })
+  )
+}
+
+function refreshPrismaClient(verbose) {
+  /** Relates to prisma/client issue, @see: https://github.com/redwoodjs/redwood/issues/1083 */
+  console.log('definitely here!!')
```
Comment (peterp) [inline packages/cli/src/commands/upgrade.js:227]:
```suggestion
```

**Review by peterp — APPROVED** [review]

File: packages/cli/src/commands/upgrade.js (hunk 1 above)
Comment (dac09) [reply packages/cli/src/commands/upgrade.js:37]: -v goes to version I believe :)

Comment (dac09) [reply packages/cli/src/commands/upgrade.js]: Nevermind, mixed it up with something else

**Review by thedavidprice — CHANGES_REQUESTED** [review]
@dac09 I suggest we go ahead and disable the PR creation and publishing workflows as well. Let me see about a way to "deactivate" vs. delete so we can still reference them.

update --> done! Thanks, DC.

**Review by thedavidprice — APPROVED** [review]

**Comment by thedavidprice** [comment]
@dac09 Title change ok?

**Pull request merged**